operator/webserver.py

```diff
@@ -41,12 +67,50 @@ def mission_stop():
             # wait for it to finish
             interfaceprocess.wait()
 
-    return {'message': 'Mission stopped'}
+    car_controls.brake = 1
+    client.setCarControls(car_controls)
+    car_controls.brake = 0 #remove brake
+
+    log = '{}: {}'.format(str(datetime.now()), 'Mission stopped.')
+    logs.append(log)
+    del logs[:]
+
+    log_file.write(log + '\n')
+    log_file.close()
+
+    return {'response': log}
 
 @app.route('/mission/reset', methods=['POST'])
```

Comment:
got this error:
```
ERROR:webserver:Exception on /mission/reset [POST]
Traceback (most recent call last):
  File "/home/sijmen/.local/lib/python2.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sijmen/.local/lib/python2.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sijmen/.local/lib/python2.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/sijmen/.local/lib/python2.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sijmen/.local/lib/python2.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "webserver.py", line 85, in mission_reset
    client.reset()
  File "../AirSim/PythonClient/airsim/client.py", line 26, in reset
    self.client.call('reset')
  File "/home/sijmen/.local/lib/python2.7/site-packages/msgpackrpc/session.py", line 41, in call
    return self.send_request(method, args).get()
  File "/home/sijmen/.local/lib/python2.7/site-packages/msgpackrpc/future.py", line 25, in get
    self.join()
  File "/home/sijmen/.local/lib/python2.7/site-packages/msgpackrpc/future.py", line 22, in join
    self._loop.start()
  File "/home/sijmen/.local/lib/python2.7/site-packages/msgpackrpc/loop.py", line 22, in start
    self._ioloop.start()
  File "/home/sijmen/.local/lib/python2.7/site-packages/tornado/ioloop.py", line 755, in start
    raise RuntimeError("IOLoop is already running")
RuntimeError: IOLoop is already running
```

Reply:
How did you get this issue? I am not able to recreate it

Reply:
So this happens when you span the reset button. It does not seem to cause any problems though